shared/chat/conversation/messages/unfurl/prompt/index.stories.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+import * as React from 'react'
+import {storiesOf} from '../../../../../stories/storybook'
+import {Box} from '../../../../../common-adapters/index'
+import UnfurlPrompt from '.'
+
+const props = {
+  domain: 'cnn.com',
+  onAlways: () => {},
```

Comment:
You should import * as Sb, then have these be `Sb.action('onAlways')`. This plumbs the buttons to the actions panel in storybook so you can see that clicking is actually working